### PR TITLE
added wat

### PR DESCRIPTION
**Pull request by MibettyMeanie**

### CODE REVIEW

File: WcaOnRails/app/jobs/sync_mailing_lists_job.rb [hunk 1]
```diff
@@ -69,6 +69,7 @@ def perform
     GsuiteMailingLists.sync_group("reports@worldcubeassociation.org", User.delegate_reports_receivers_emails)
     GsuiteMailingLists.sync_group("advisory@worldcubeassociation.org", Team.wac.current_members.includes(:user).map(&:user).map(&:email))
     GsuiteMailingLists.sync_group("sports@worldcubeassociation.org", Team.wsot.current_members.includes(:user).map(&:user).map(&:email))
+    GsuiteMailingLists.sync_group("archive@worldcubeassociation.org ", Team.wat.current_members.includes(:user).map(&:user).map(&:email))
```
Comment (gregorbg) [inline WcaOnRails/app/jobs/sync_mailing_lists_job.rb:72]: Nitpick: There's a superfluous space at the end of the email address

**Review by gregorbg — CHANGES_REQUESTED** [review]
Thanks, in general the structure is fine :D

Unfortunately, you made a very specific typo that could break the Mailing List Sync, so please fix that before we can proceed with merging

File: WcaOnRails/app/jobs/sync_mailing_lists_job.rb (hunk 1 above)
Comment (gregorbg) [reply WcaOnRails/app/jobs/sync_mailing_lists_job.rb:72]: This typo also explains the current test failure

Comment (MibettyMeanie) [reply WcaOnRails/app/jobs/sync_mailing_lists_job.rb]: got it thanks!

File: WcaOnRails/db/migrate/20220516124717_add_team_for_wat.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddTeamForWat < ActiveRecord::Migration[6.1]
+  def change
+    Team.create(friendly_id: 'wat', email: "archive@worldcubeassociation.org")
```
Comment (gregorbg) [inline WcaOnRails/db/migrate/20220516124717_add_team_for_wat.rb:5]: Please create in the initial migration as `hidden: true`, similar to https://github.com/thewca/worldcubeassociation.org/pull/6900/files#diff-5d6622d6d46140624e4da2f52867db2f36420549b50a2ca2defdf0fb8377a846R5

**Review by gregorbg — CHANGES_REQUESTED** [review]
One tiny concern about creating the team as `hidden` in the initial migration, sorry I didn't catch that before :(

File: WcaOnRails/db/migrate/20220516124717_add_team_for_wat.rb (hunk 2 above)
Comment (MibettyMeanie) [reply WcaOnRails/db/migrate/20220516124717_add_team_for_wat.rb]: done

**Review by gregorbg — APPROVED** [review]

**Comment by gregorbg** [comment]
(note: Feel free to merge this yourself for a well-deserved dose of satisfaction! Otherwise I will probably do so over the weekend to keep our backlog as short as possible)

**Pull request merged**